src/bootstrap/config.rs

```diff
@@ -1423,6 +1427,18 @@ impl Config {
     pub fn submodules(&self, rust_info: &GitInfo) -> bool {
         self.submodules.unwrap_or(rust_info.is_git())
     }
+
+    /// Return whether we're using an externally provided LLVM.
+    ///
+    /// In particular, this may not be the latest version, nor contain our rust-specific patches.
+    pub fn using_external_llvm(&self, target: TargetSelection) -> bool {
```

Comment:
Can we unify with `fn is_rust_llvm` in Build?

Reply:
Didn't know that existed, thanks :)